File: 122-best-time-to-buy-and-sell-stock-ii/122-best-time-to-buy-and-sell-stock-ii.cpp
class Solution {
public:
    int maxProfit(vector<int>& prices) {
        
        int mp=0;
        int n=prices.size();
        for(int i=0;i<n-1;i++)
        {
            int t=0;
            if(prices[i+1]>prices[i])
            mp+=(prices[i+1]-prices[i]);
        }
        return mp;
    }
};

File: 122-best-time-to-buy-and-sell-stock-ii/122-best-time-to-buy-and-sell-stock-ii.java
class Solution {
    public int maxProfit(int[] p) {
        int pro=0;
        for(int i=0;i<p.length-1;i++){
            if(p[i+1]>p[i]){
                pro+=(p[i+1]-p[i]);
            }
        }
        return pro;
    }
}